examples/README.md

```diff
@@ -421,19 +360,20 @@ Add the following content to the function, `helloworld_test()` in test.sh.  Plea
 ```
 helloworld_test() {
     say "- do hello world"
-    # install and instantiate helloworld  chaincode
 
-    # builds the docker image; creates the docker container and enclave;
+    # install helloworld  chaincode
     # input:  CC_ID:chaincode name; CC_VERS:chaincode version;
     #         ENCLAVE_SO_PATH:path to build artifacts
     say "- install helloworld chaincode"
-    try ${PEER_CMD} chaincode install -l fpc-c -n ${CC_ID} -v ${CC_VERS} -p ${ENCLAVE_SO_PATH}
-    sleep 3
+    PKG=/tmp/${CC_ID}.tar.gz
+    try ${PEER_CMD} lifecycle chaincode package --lang ${CC_LANG} --label ${CC_ID} --path ${CC_PATH} ${PKG}
```

Comment:
CC_PATH is not defined? Neither is CC_LANG?

Reply:
Good catch! will fix that